src/testlib/test/yamlToBson_test.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019-present MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <ios>
+#include <string>
+
+#include <yaml-cpp/yaml.h>
+
+#include <bsoncxx/json.hpp>
+
+#include <iostream>
+#include <testlib/helpers.hpp>
+#include <testlib/yamlToBson.hpp>
+
+namespace genny::testing {
+
+using Catch::Matchers::Matches;
+
+namespace {
```

Comment:
C++ unit tests should be entirely put into an anonymous namespace because there's nothing defined within them that requires external linkage.

Reply:
The tests themselves aren't in the anonymous namespace only helper functions are.

Reply:
I mis-read your comment - good call - moved the tests themselves to the anon namespace as well.

Reply:
Gotta save those microseconds on link times!